plugins/out_stackdriver/stackdriver.c

```diff
@@ -2737,6 +2763,11 @@ static struct flb_config_map config_map[] = {
       0, FLB_TRUE, offsetof(struct flb_stackdriver, task_id),
       "Set the resource task id"
     },
+    {
+     FLB_CONFIG_MAP_STR, "compress", NULL,
+     0, FLB_FALSE, 0,
```

Comment:
I think your indentation is off…
```suggestion
      FLB_CONFIG_MAP_STR, "compress", NULL,
      0, FLB_FALSE, 0,
```